test/e2e/daemon_restart.go

```diff
@@ -241,6 +241,9 @@ var _ = Describe("DaemonRestart", func() {
 
 	It("Scheduler should continue assigning pods to nodes across restart", func() {
 
+		//TODO: check why this test doesn't work on GKE.
```

Comment:
I think that we know it  - it's basically because experimental api is disabled in GKE and daemons are part of experimental api.
Same below.

FYI: @brendandburns @roberthbailey